### DISCUSSION TITLE
[@types/yargs] Missing showVersion type on Argv interface

### DISCUSSION THREAD

**Discussion by jdecked** [DefinitelyTyped/DefinitelyTyped#57145] [Issues with a @types package] [answered]
The `Argv` interface is missing the `showVersion` property.

[Here is a link to showVersion in the docs](http://yargs.js.org/docs/#api-reference-showversionconsolelevel-printcallback), where it's noted as a valid property.

[Here is showVersion's definition in yargs.](https://github.com/yargs/yargs/blob/e9379e27d49820f4db842f22cda6410bbe2bff10/lib/yargs-factory.ts#L1311)

Here is a command file that fails to typecheck:
```
import { Argv } from 'yargs';
import MyFancyLogger from './logger';

export const command = 'version';
export const describe = 'Show version number';

export function builder(yargs: Argv) {
  const log = new MyFancyLogger();
  return yargs.showVersion((version: string) => log.info(version));
  // Error: Property 'showVersion' does not exist on type 'Argv<{}>'.
}

export function handler() {}
```

`yargs` version: `^17.2.1`
`@types/yargs` version: `^17.0.5`

Happy to PR this if it's an unintentional omission.

[a comment by an automated account is not shown]

**Comment by forivall** [comment, marked as answer]
If it's in the docs, it's valid! PR is welcome.

**Reply by jdecked** [reply]
Super, thanks! PR is here: #57178
